themes/squadtheme/static/css/site.css

```diff
@@ -16,6 +16,11 @@ h1 a {
     text-decoration: None;
 }
 
+a img {
+    height: 80px;
+    width: 80px;
```

Comment:
My issue with vw or vmin is that things start to look pretty bad on mobile, but I guess we can deal with that using an @media thing.

Reply:
Yeah that's a good way to handle it. Also the whole site is fully based right now on the viewport so if we want to change that I'd rather do it systematically rather than have a few ad hoc differences.